includes/modules/shipping_estimator.php

```diff
@@ -209,7 +209,6 @@ function shipincart_submit() {
                         foreach ($selected_quote[0]['methods'] as $qkey => $qval) {
                             if ($qval['id'] == $method) {
                                 $selected_quote[0]['methods'] = [$qval];
-                                continue;
```

Comment:
Should that have been `continue 2;` instead? ie: abort the current foreach (because it's satisfied) and jump to the next iteration of the outer foreach. Perhaps the redundant continue statement was a bug that should have specified the '2' part?

Reply:
Other than that question, I'm satisfied with these formatting updates.

Reply:
I've added the continue in back in for now so this PR does not touch functionality. I'll look at the continue shortly, separately.